rotkehlchen/db/dbhandler.py

```diff
@@ -1424,6 +1425,10 @@ def remove_single_blockchain_accounts(
         if blockchain in SUPPORTED_BITCOIN_CHAINS:
             for address in accounts:
                 self.delete_data_for_bitcoin_address(write_cursor, address, blockchain)  # type: ignore  # mypy doesn't understand the blockchain if check
+        if blockchain == SupportedBlockchain.SOLANA:
```

Comment:
shuldn't all these ifs be if/elif/elif/else? Or perhaps a `match` statement? Cause right now you are doing multiple conditional checks while this is not necessary.